sonar-db/src/test/resources/org/sonar/db/version/v60/MakeUuidColumnNotNullOnSnapshotsTest/in_progress_snapshots.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE "SNAPSHOTS" (
+  "ID" INTEGER NOT NULL GENERATED BY DEFAULT AS IDENTITY (START WITH 1, INCREMENT BY 1),
+  // NULLABLE
```

Comment:
FYI `#` fails but not `//` ! It seems to be a bug in H2 when comment is inside request (`org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement`)